README.rst

```diff
@@ -153,6 +153,7 @@ To contribute assets to Aspects:
 #. The script will also warn about missing `_roles` in dashboards. Superset does not export
    these, so you will need to manually add this key with the roles that are necessary to
    view the dashboard. See the existing dashboards for how this is done.
+#. Re-build your openedx-assets image with `tutor images build openedx-assets --no-cache`
```

Comment:
```suggestion
#. Re-build your openedx-assets image with `tutor images build superset --no-cache`
```

Reply:
My bad, but is `aspects-superset`, right?

Reply:
yes